### PR TITLE
Add prerequisites instructions

### PR DESCRIPTION
**Pull request by roipoussiere**
Add instruction on the tutorial to invite user to read prerequisites instruction.

That fixes installation issues like #697.

## PR Checklist:

- **All new features have been tested**: inapplicable
- **All new features have been documented**: inapplicable
- [x] **I have read the CONTRIBUTING.md file**: so I came across [this page](https://beeware.org/contributing/how/) that explained me to just send a PR so I think I'm good
- **I will abide by the code of conduct**: not found :/

### CODE REVIEW

File: docs/tutorial/tutorial-0.rst [hunk 1]
```diff
@@ -11,7 +11,8 @@ button, that prints to the console when you press the button.
 Set up your development environment
 ===================================
 
-Open a command prompt on your computer and make sure that you can successfully run the :code:`python3` command. Create a working directory for your code and change to it.
+Make sure you installed `Toga prerequisites <https://github.com/beeware/toga#prerequisites>`_, such as Python3 and other librariesn the create a working directory for your code and change to it.
```
Comment (danyeaw) [inline docs/tutorial/tutorial-0.rst]: > Make sure you installed `Toga prerequisites <https://github.com/beeware/toga#prerequisites>`_, such as Python3 and other librariesn the create a working directory for your code and change to it.

Make sure you installed the `Toga prerequisites <https://github.com/beeware/toga#prerequisites>`_, such as Python 3 and the other libraries. Then create a working directory for your code and change to it.

File: docs/tutorial/tutorial-0.rst [hunk 2]
```diff
@@ -75,6 +76,8 @@ Next, install Toga into your virtual environment:
 
       (venv) C:\...>pip install --pre toga
 
+If you get other errors, please check that you followed [prerequisites](https://github.com/beeware/toga#prerequisites) instructions.
```
Comment (danyeaw) [inline docs/tutorial/tutorial-0.rst]: This doesn't look like reStructured Text for the external link, did you accidentally use Markdown instead?

Comment (roipoussiere) [reply docs/tutorial/tutorial-0.rst]: indeed.

File: docs/tutorial/tutorial-0.rst (hunk 1 above)
Comment (roipoussiere) [reply docs/tutorial/tutorial-0.rst]: Oops!

File: docs/tutorial/tutorial-0.rst [hunk 3]
```diff
@@ -75,6 +76,8 @@ Next, install Toga into your virtual environment:
 
       (venv) C:\...>pip install --pre toga
 
+If you get other errors, please check that you followed `prerequisites <https://github.com/beeware/toga#prerequisites>`_ instructions.
```
Comment (danyeaw) [inline docs/tutorial/tutorial-0.rst]: One more update: please check that you followed *the* *prerequisite* instructions.

Comment (roipoussiere) [reply docs/tutorial/tutorial-0.rst]: It is not *the prerequisite* (without the **s**)?

**Comment by phildini** [comment]
Hi there! It looks like this PR might be dead, so we're closing it for now. Feel free to re-open it if you'd like to continue. 😄

**Comment by roipoussiere** [comment]
> Hi there! It looks like this PR might be dead, so we're closing it for now. Feel free to re-open it if you'd like to continue.

I was waiting for a last answer from @danyeaw in a conversation but anyway I just resolved it so I think it can be re-opened and merged.

File: docs/tutorial/tutorial-0.rst (hunk 3 above)
Comment (roipoussiere) [reply docs/tutorial/tutorial-0.rst:79]: fixed: https://github.com/roipoussiere/toga/commit/b36a8bfadc7a63248c9c53cf025c5d5540ee55e0

**Pull request merged**